### ISSUE TITLE
Feature Request - check usernames for whitespace

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#7802]
**Is your feature request related to a problem? Please describe.**

There was an attempt to create an operator with whitespace in the username.

**Describe what you're trying to accomplish** 

Prevent frustration

**Describe the solution you'd like**

Stop this earlier

**Describe alternatives you've considered**

Anger

**Additional context**

nope

**Priority**

high

**Comment by dustymc** [comment]
Expansion: Check agents for https://github.com/ArctosDB/arctos/issues/7649#issuecomment-2111250745 during invitation, duplicates with accounts is not acceptable. Maybe require AWG signoff if an agent looks like a duplicate????

**Comment by dustymc** [comment]
/component/api/agent:isAcceptableOperator exists but needs refinement

**Comment by dustymc** [comment]
There are now additional checks on operators at the time of creation, I hope it's all reasonable and has no bugs, and I am sorely missing a public dev box on which I could request testing @mkoo ....
